python/pyspark/sql/readwriter.py

```diff
@@ -627,8 +627,6 @@ def jdbc(self, url, table, column=None, lowerBound=None, upperBound=None, numPar
 
         Parameters
         ----------
-        url : str
-            a JDBC URL of the form ``jdbc:subprotocol:subname``
         table : str
             the name of the table
         column : str, optional
```

Comment:
I think we can remove `lowerBound`, `upperBound`, and `numPartitions`.
And, fix the description of `column` to something like:

Alias of `partitionColumn` option. Refer to `partitionColumn` in `Data Source Option <...>`_ in the version you use.